.golangci.yml

```
@@ -0,0 +1,50 @@
+
+issues:
+  max-issues-per-linter: 0
+  max-same-issues: 0
+
+severity:
+  default: error
+
+output:
+  formats:
+    text:
+      path: stdout
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)